checks.d/marathon.py

```diff
@@ -56,6 +56,10 @@ def check(self, instance):
                     if attr in app:
                         self.gauge('marathon.' + attr, app[attr], tags=tags)
 
+        response = self.get_json(urljoin(url, "v2/deployments"), timeout, auth)
```

Comment:
Do you think we can get this behind a config flag? I imagine this will slow down the check a bit, being not a Marathon expert was wondering if this is something any Marathon user would trade for the new metric or not.